Xamarin.Forms.Core/ThicknessTypeConverter.cs

```diff
@@ -3,6 +3,7 @@
 
 namespace Xamarin.Forms
 {
+	[Xaml.ProvideCompiled("Xamarin.Forms.Core.XamlC.ThicknessTypeConverter")]
```

Comment:
it references the class by name, so the Core doesn't hold a reference on the BuildTask assembly